easystacks/software.eessi.io/2023.06/icelake_cclake/057-eb-4.9.2.yml

```
@@ -3,4 +3,177 @@ easyconfigs:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Pint/0.24-GCCcore-13.2.0/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Boost.MPI/1.83.0-gompi-2023b/easybuild/Boost.MPI-1.83.0-gompi-2023b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Boost.MPI/1.83.0-gompi-2023b/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/ESPResSo/4.2.2-foss-2023b/easybuild/ESPResSo-4.2.2-foss-2023b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/ESPResSo/4.2.2-foss-2023b/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/pyMBE/0.8.0-foss-2023b/easybuild/pyMBE-0.8.0-foss-2023b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/pyMBE/0.8.0-foss-2023b/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/KaHIP/3.16-gompi-2023a/easybuild/KaHIP-3.16-gompi-2023a.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/KaHIP/3.16-gompi-2023a/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/OpenFOAM/v2312-foss-2023a/easybuild/OpenFOAM-v2312-foss-2023a.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/OpenFOAM/v2312-foss-2023a/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/LLVM/14.0.6-GCCcore-12.3.0-llvmlite/easybuild/LLVM-14.0.6-GCCcore-12.3.0-llvmlite.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/LLVM/14.0.6-GCCcore-12.3.0-llvmlite/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/numba/0.58.1-foss-2023a/easybuild/numba-0.58.1-foss-2023a.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/numba/0.58.1-foss-2023a/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/librosa/0.10.1-foss-2023a/easybuild/librosa-0.10.1-foss-2023a.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/librosa/0.10.1-foss-2023a/easybuild/reprod/easyblocks/*.py
```

librosa failed due to the same issue as we saw in https://github.com/EESSI/software-layer/pull/897#issuecomment-2614919963.

```suggestion
  - librosa-0.10.1-foss-2023a.eb:
      options:
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21434
        from-commit: 4b6170603150c9c93602b0deb726869d19a390f0
        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/librosa/0.10.1-foss-2023a/easybuild/reprod/easyblocks/*.py
```